src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -198,6 +198,9 @@ std::vector<std::string> disabledTestPatterns() {
     // Issue: 122177
     retVector.emplace_back(R"(.*smoke_LSTMCellCommon/LSTMCellTest.Inference.*_modelType=f16.*)");
     retVector.emplace_back(R"(.*smoke_LSTMSequenceCommonZeroClip/LSTMSequenceTest.Inference.*_modelType=f16.*)");
+    // Issue 122699
```

Comment:
to use #ifdef to disable only on macos arm64?

Reply:
It's already under `#if defined(__APPLE__) && defined(OPENVINO_ARCH_ARM64)` (https://github.com/openvinotoolkit/openvino/pull/20451/files#diff-a8b7858ccc95cb4e158fe096691dc8c080d8aed4003555a869cadadf4fe55f51R193)